.github/workflows/cloud_csharp_client_test.yaml

```diff
@@ -0,0 +1,100 @@
+      branch_name:
+        description: Name of the branch to test client from
+        required: true
+        default: master
+      base_url:
+        description: Base url of the cloud env. I.e https://uat.hazelcast.cloud
+        required: true
+        default: https://uat.hazelcast.cloud
+      hzVersion:
+        description: Version of hazelcast (For cloud standard cluster, it has different versions, i.e 5.0.2-2)
+        required: true
+
+jobs:
+  test_standard_cluster:
+    runs-on: ubuntu-latest
+    name: Hazelcast cloud standard cluster tests with csharp
+    steps:
+      - name: Checkout to scripts
+        uses: actions/checkout@v2
+        
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+          
+      - name: Setup Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: 8
+          
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: |
+            3.1.x
+            5.0.x
+            6.0.x
+          
+      - name: Checkout to ${{ github.event.inputs.branch_name }}
+        uses: actions/checkout@v2
+        with:
+          repository: ${{ github.event.inputs.organization_name }}/hazelcast-csharp-client
+          path: client
+          ref: ${{ github.event.inputs.branch_name }}
+        
+      - name: Copy reqired files to under test folders
+        run: |
+          rm -rf $GITHUB_WORKSPACE/client/src/Hazelcast.Net.Testing/Remote
+          cp -R $GITHUB_WORKSPACE/HazelcastCloudTests/csharphazelcastcloudtests/Remote $GITHUB_WORKSPACE/client/src/Hazelcast.Net.Testing/Remote/
+          cp -R $GITHUB_WORKSPACE/HazelcastCloudTests/csharphazelcastcloudtests/CloudTests $GITHUB_WORKSPACE/client/src/CloudTests/
+          ls $GITHUB_WORKSPACE/client/src/CloudTests
+         
+      - name: Start RC for UAT
+        if: ${{ github.event.inputs.base_url == 'https://uat.hazelcast.cloud' }}
+        env:
+          baseUrl: ${{ github.event.inputs.base_url }} 
+          apiKey: ${{ secrets.APIKEY_UAT }}
+          apiSecret: ${{ secrets.APISECRET_UAT }}
+        run: |
+          python start_remote_controller.py &> rc_standard_cluster.log &
+          sleep 30
+          
+      - name: Start RC for PROD
+        if: ${{ github.event.inputs.base_url == 'https://cloud.hazelcast.com' }}
+        env:
+          baseUrl: ${{ github.event.inputs.base_url }} 
+          apiKey: ${{ secrets.APIKEY_PROD }}
+          apiSecret: ${{ secrets.APISECRET_PROD }}
+        run: |
+          python start_remote_controller.py &> rc_standard_cluster.log &
+          sleep 30
+          
+      - name: Build project
+        run: |
+          dotnet build -c Release
+        working-directory: client/src/CloudTests
+                   
+      - name: Run cloud tests
+        env: 
+          baseUrl: ${{ github.event.inputs.base_url }}
+          hzVersion: ${{ github.event.inputs.hzVersion }}
```

Comment:
These are environment variables, right? I believe it makes sense to use them as capitalized https://google.github.io/styleguide/shellguide.html#constants-and-environment-variable-names

Reply:
Yes, it's better to follow conventions but it's a bit deep. Already used and done in many places including other client tests.